interlok-core/src/main/java/com/adaptris/core/CoreConstants.java

```diff
@@ -39,6 +41,23 @@
    * </p>
    */
   public static final String STOP_PROCESSING_KEY = "adpstopprocessing";
+  
+  /**
+   * <p>
+   * Metadata value which determines whether or not to stop processing additional services and/or producers
+   * </p>
+   *
+   * @see #STOP_PROCESSING_KEY
+   * @see #KEY_WORKFLOW_SKIP_PRODUCER
+   */
+  public static final String STOP_PROCESSING_VALUE = "true";
+  
+  /**
+   * A simply function check to test if the processing of the given message should stop.
+   */
+  public static final Function<AdaptrisMessage, Boolean> shouldStopProcessing = adaptrisMessage -> {
+    return STOP_PROCESSING_VALUE.equals(adaptrisMessage.getMetadataValue(STOP_PROCESSING_KEY));
+  };
```

Comment:
While I do like a `{}` block; is it not possible to simplify this to just
```
public static final Function<AdaptrisMessage, Boolean> shouldStopProcessing = 
  am -> BooleanUtils.toBoolean(am.getMetadataValue(STOP_PROCESSING_KEY))
```

Then we can just deprecate the STOP_PROCESSING_VALUE and say it can be any of `y|yes|on|true` since BooleanUtils does the case magic for us as well...